osbs/utils.py

```diff
@@ -361,7 +362,7 @@ def make_name_from_git(repo, branch, limit=53, separator='-', hash_size=5):
 
     repo = git_repo_humanish_part_from_uri(repo)
     branch = branch or 'unknown'
-    full = repo + branch
+    full = urlparse(repo).path.lstrip('/') + branch
```

Comment:
This looks odd. At this point, `repo` came from `git_repo_humanish_part_from_uri()`, which returns the result from a call to `os.path.basename()`. Why would there be '/' characters at the beginning of that?

Reply:
Fixed.